packages/icon/test/mwc-icon.test.ts

```diff
@@ -33,4 +33,16 @@ suite('mwc-icon', () => {
   test('initializes as an mwc-icon', () => {
     assert.instanceOf(element, Icon);
   });
+
+  test('sets icon color', () => {
+    element.style.color = "blue";
```

Comment:
Thanks for adding to our test suite @onelittlekiwi !

If you want to address the lint errors, we might be able to merge this our base